conf/defaults.ini

```diff
@@ -295,6 +295,14 @@ content_security_policy_template = """script-src 'self' 'unsafe-eval' 'unsafe-in
 # Controls if old angular plugins are supported or not. This will be disabled by default in Grafana v9.
 angular_support_enabled = true
 
+[security.encryption]
```

Comment:
I think we're already using this section for Enterprise features, but it shouldn't cause any conflict, right? 🤔

Reply:
I think it's fine.

Reply:
Yeah, I did some tests and apparently the `gopkg.in/ini.v1` package is able to automatically merge fields from same section. 

So, something like:
```
[security.encryption]
data_keys_cache_ttl = 15m

data_keys_cache_cleanup_interval = 1m

# Enterprise only
[security.encryption]
algorithm = "rsa-sha256"
```

Ends up with the `security.encryption` initialized with these three values.

Is that what you meant? @xlson  🤔  Thanks!!

Reply:
That's what I meant, great! I'll mark this conversation as resolved then.